docs/src/InsolationExamples.md

```diff
@@ -24,16 +24,19 @@ diurnal_cycle(lat, lon, date, timezone, "Finland_June.png")
 ## Daily-mean insolation
 ### Insolation in J2000
 ```@example
-using CLIMAParameters
-using CLIMAParameters.Planet
-struct EarthParameterSet <: AbstractEarthParameterSet end
-const param_set = EarthParameterSet()
+import CLIMAParameters as CP
+import Insolation
+import Insolation.Parameters as IP
+
+FT = Float64
```

Comment:
The package should now work also with Float32. If that is what ClimaAtmos will use it might be best to use that in the docs too.

Reply:
It should work fine with Float32. This example was previously using Float64, so I kept it unchanged. I'm just declaring Float64 here so that it's transparent (`create_insolation_parameters` requires the float type in order to make the new parameter structs).